packages/bun/src/index.ts

```diff
@@ -76,6 +76,11 @@ export {
   startSpanManual,
   continueTrace,
   metrics,
+  functionToStringIntegration,
+  inboundFiltersIntegration,
+  linkedErrorsIntegration,
+  moduleMetadataIntegration,
+  requestDataIntegration,
```

Comment:
Do we want to export these here? Previously, these were not accessible directly from `@sentry/bun`.

Reply:
OK, I guess we may move the `moduleMetadatIntegation` to browser in v8? 🤔 if this is browser-only, no need to have & export it from core. But we can see!

Reply:
or actually, maybe I move it there right away 🤔